cpp/src/gandiva/function_registry_string.cc

```diff
@@ -88,6 +88,10 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
                      "gdv_fn_initcap_utf8",
                      NativeFunction::kNeedsContext | NativeFunction::kCanReturnErrors),
 
+      NativeFunction("elt", {}, DataTypeVector{int32(), utf8()}, utf8(),
+                     kResultNullIfNull, "gdv_fn_elt_utf8",
+                     NativeFunction::kNeedsContext | NativeFunction::kCanReturnErrors),
```

Comment:
Why does this need the flags kNeedsContext and kCanReturnErrors?

Reply:
It can return an error if it could not alocate the memory for the output